pkg/nsx/services/ipaddressallocation/ipaddressallocation.go

```diff
@@ -41,10 +41,10 @@ func InitializeIPAddressAllocation(service common.Service, vpcService common.VPC
 		BindingType: model.VpcIpAddressAllocationBindingType(),
 	}}
 
-	tags := []model.Tag{
-		{Scope: String(common.TagScopeIPAddressAllocationCRUID)},
-	}
-	go ipAddressAllocationService.InitializeResourceStore(&wg, fatalErrors, ResourceTypeIPAddressAllocation, tags, ipAddressAllocationService.ipAddressAllocationStore)
```

Comment:
We still need at least cluster name tag to filter.
And please make sure GC would not delete other IPAddressAllocation for LB if you put them also into store. Another option is to search tags again in Cleanup() instead of using store.

Reply:
ok